packages/oui-navbar/src/group/navbar-group.service.js

```diff
@@ -0,0 +1,63 @@
+export default class {
+    constructor ($timeout, NavbarService, KEYBOARD_KEYS) {
+        "ngInject";
+
+        this.$timeout = $timeout;
+        this.navbarService = NavbarService;
+        this.KEYBOARD_KEYS = KEYBOARD_KEYS;
+
+        this.keyboardNav = {};
+    }
+
+    addItemToGroup (item, groupName) {
+        // Create the group if it doesn't exist
+        if (angular.isUndefined(this.keyboardNav[groupName])) {
+            this.keyboardNav[groupName] = [];
+        }
+
+        // Add item to the group
+        this.keyboardNav[groupName].push(item);
+    }
+
+    bindGroup (groupName) {
+        const keys = {};
+        const keysRegex = new RegExp([
+            this.KEYBOARD_KEYS.ALT,
```

Comment:
Later it will be cleaner to use this for the other components ^^

Reply:
Yes, i think we have the same use for oui-dropdown :)